AmericanMedicalAssociation/ama-style-guide-2
# Remove 'atom' and 'molecule' VRTs

## Ticket(s)
N/A

## Description
Removed backstop test cases for atoms and molecules. This will cut down travis run times considerably for SG2.


## To Test
- Pull branch 'bugfix/remove-atom-vrt-tests' and run 'npm i' to update dependencies.
- Run backstop tests and confirm atoms and molecules are not part of results.

## Visual Regressions
A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/bugfix/remove-atom-vrt-tests/html_report/index.html).

For more information on visual regression testing review the [How do I run tests?](https://issues.ama-assn.org:8446/confluence/pages/viewpage.action?pageId=23298568) document in Confluence.

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
